packages/next-templates/src/app/meldingen/verhuizing-doorgeven/page.tsx

```diff
@@ -0,0 +1,398 @@
+'use client';
+
+import {
+  Article,
+  AccordionProvider,
+  ButtonLink,
+  ButtonGroup,
+  Document,
+  Heading1,
+  Heading2,
+  Heading3,
+  Link,
+  Paragraph,
+  Page,
+  PageContent,
+  SpotlightSection,
+  UnorderedList,
+  UnorderedListItem,
+} from '@utrecht/component-library-react';
+import '@/app/styling/css/detail-page.css';
+import '@utrecht/design-tokens/dist/index.css';
+import { ExampleHeaderFunnel } from '@/components/ExampleHeader/ExampleHeaderFunnel/ExampleHeaderFunnel';
+import { ExampleFooterFocus } from '@/components/ExampleFooter/ExampleFooterFocus/ExampleFooterFocus';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { UtrechtDigidLogo, UtrechtIconArrow } from '@utrecht/web-component-library-react';
+import { useForm } from 'react-hook-form';
+import { useRouter } from 'next/navigation';
+import BacklinkIcon from '@/app/styling/assets/backlink-icon.svg';
+
+export default function Home() {
+  const router = useRouter();
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<{ [key: string]: string }>();
+
+  return (
+    <Document>
+      <Page>
+        <ExampleHeaderFunnel />
+        <ExampleNavigation />
+        <PageContent>
+          <Article className="voorbeeld-article-space">
+            <Link className="voorbeeld-link-back" href="./">
+              <BacklinkIcon />
+              <span className="voorbeeld-span--underlined">Terug</span>
+            </Link>
+            <Heading1 className="voorbeeld-heading-1">Verhuizing doorgeven</Heading1>
+            <Paragraph>
+              Als u verhuist naar Utrecht of binnen Utrecht, geeft u uw adres aan ons door. Verhuist u van Utrecht naar
+              een andere gemeente? Geef dan uw verhuizing door in uw nieuwe gemeente.
+            </Paragraph>
+
+            <Link href="https://www.utrecht.nl/city-of-utrecht/moving-to-utrecht-or-moving-within-utrecht">
+              <UtrechtIconArrow />
+              Read this information in English
+            </Link>
```

Comment:
Een `<Link>` is een `display: inline` element, `<Paragraph>` is een `display: block` element. Het is gebruikelijk (en ik vind het netjes) om niet op deze manier inline en block elementen af te wisselen als "siblings". Is er niet een element dat we netjes om de link heen kunnen zetten?